pkg/approver/constraints/validation.go

```diff
@@ -66,12 +66,12 @@ func (c Constraints) Validate(_ context.Context, policy *policyapi.CertificateRe
 		}
 
 		maxSize := consts.PrivateKey.MaxSize
-		if maxSize != nil && (*maxSize <= 0 || *maxSize > 8192) {
+		if maxSize != nil && (*maxSize < 0 || *maxSize > 8192) {
```

Comment:
Question: would there be any reason for a constraint with a 0 max key size?

Reply:
Yes, this makes a robust "deny-all" policy possible.https://github.com/cert-manager/approver-policy/blob/bde0903a2c66719651c41c4348207ff7621ddf15/docs/examples/deny-all.yaml